lasso/emme.py

```diff
@@ -268,145 +258,147 @@ def create_emme_network(
         setup = SetupEmme(model_tables, out_dir, _NAME, include_transit, parameters)
         setup.run()
 
-def prepare_table_for_taz_drive_network(
-    nodes_df,
-    links_df,
-    parameters,
-):
-
-    """
-    prepare model table for taz-scale drive network, in which taz nodes are centroids
-    keep links that are drive_access == 1 and assignable
 
-    Arguments:
-        nodes_df -- node database
-        links_df -- link database
+def extract_bus_shapes(transit_network: StandardTransit, route_type_bus_id:int = 3):
     
-    Return:
-        dictionary of model network settings
-    """
-
-    model_tables = dict()
-
-    # use taz as centroids, drop maz nodes and connectors
-    model_tables["centroid_table"] = nodes_df[
-        nodes_df.N.isin(parameters.taz_N_list)
-    ].to_dict('records')
-
-    model_tables["connector_table"] = links_df[
-        (links_df.A.isin(parameters.taz_N_list)) | (links_df.B.isin(parameters.taz_N_list))
-    ].to_dict('records')
-
-    drive_links_df = links_df[
-        ~(links_df.A.isin(parameters.taz_N_list + parameters.maz_N_list)) & 
-        ~(links_df.B.isin(parameters.taz_N_list + parameters.maz_N_list)) &
-        ((links_df.drive_access == 1) & (links_df.assignable == 1))
-    ].copy()
-
-    model_tables["link_table"] = drive_links_df.to_dict('records')
-
-    drive_nodes_df = nodes_df[
-        (nodes_df.N.isin(drive_links_df.A.tolist()) + nodes_df.N.isin(drive_links_df.B.tolist()))
-    ].copy()
-
-    model_tables["node_table"] = drive_nodes_df.to_dict('records')
-
-    return model_tables
+    shapes = transit_network.feed["shapes"].copy()
+    trips = transit_network.feed["trips"].copy()
+    routes = transit_network.feed["routes"].copy()
```

Comment:
As I previously commented, please try naming objects with their types, e.g., `shapes_df` instead of `shapes`

Reply:
fixed in df424f9